packages/visualizations/src/components/Table/Headers/Th.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+    import type { DataFormat } from '../types';
+    import { stickyColumnsWidth, stickyColumnsOffset, lastStickyColumn } from '../store';
+
+    export let key: string;
+    export let dataFormat: DataFormat;
+    export let isHorizontallyScrolled: boolean;
+    let clientWidth: number;
+
+    // Only updates columns that have been initialized after a reset in Table.svelte
+    $: if ($stickyColumnsWidth.has(key)) {
+        stickyColumnsWidth.updateColumn(key, clientWidth);
+    }
+</script>
+
+<th
+    style={`--sticky-offset: ${$stickyColumnsOffset.get(key)}px`}
+    class={`table-header--${dataFormat}`}
+    class:sticky={$stickyColumnsWidth.has(key)}
+    class:isLastSticky={key === $lastStickyColumn}
+    class:isHorizontallyScrolled
```

Comment:
These three classes are shared between `Th` and `Cell`
What do you think to create a hook to build these sticky classes ?